laravel/app/Console/Commands/PushSchedule.php

```diff
@@ -161,13 +156,39 @@ public function handle()
         $spreadsheetId = '1npNBs_j6BvmZO29GHlEJ-mROGhtBEqM7_KNKdAnNLxY';
         $title = '\''.$title.'\'!';
         $range = $title.'A1:M15';
```

Comment:
I may be misremembering, but I thought we decided changing `range` to be dynamic/calculated (rather than hardcoded, as it is now) was in scope for this PR?

Reply:
I don't think we talked about that being in scope. I could take a look but it would be a little harder to test and I may want to have access to the google sheet (if we don't have it already).

Reply:
I don't remember this being in scope either but if you want to have it in this PR that's alright. @Spensenaz you might find this useful 
   [https://www.geeksforgeeks.org/find-excel-column-name-given-number/](url)

Reply:
Okay so I'll translate that algorithm to php, the only reason I'd want access to the Google Sheet is to make sure it still works once we implement it. For right now I'll just test the string and that should be fine.

Reply:
Sounds good to me. Re-request my review once you've got it implemented!

Also, David mentioned making the spreadsheet available in the REMODEL google drive folder. If it's not there, it might be worth reaching out and reminding him.

Reply:
Update: We discussed elsewhere and decided to get this merged for now, and will open a separate PR for dynamic range.